Provide Makefile target for removing example apps (#407)
vintasoftware/django-react-boilerplate: README.md

`pip-compile requirements.in > requirements.txt` or `make upgrade`
`pip install -r requirements.txt`

### Cleaning example code

> Before start creating you own apps, run the command `make clean_examples` in order to clean the example apps from the front and backend. It will also remove it's usage.

Reviewer: There are a few typos here :)

"Before start creating **your** own apps, run the command `make clean_examples` in order to clean **up** the example apps from the front and backend. It will also remove **its** usage."

What do you mean by "it will also remove its usage", btw?

Author: done.

"it will also remove its usage" does not make sense anymore. It has been removed.

## Deployment 
### Setup

> This project comes with a `app.json` file for heroku, that can be used to create an app on heroku from a GitHub repository.

Reviewer: I know this is not part of the issue, but could you please replace "a `app.json`" by "an `app.json`" and "that can" by "which can"? Thanks!

Author: done

Reviewer: I think you've missed the that -> which part :P

`pip-compile requirements.in > requirements.txt` or `make upgrade`
`pip install -r requirements.txt`

### Cleaning example code
Before start creating your own apps, run the command `make clean_examples` in order to clean up the example apps from the front and backend.

## Deployment 
### Setup

> This project comes with an `app.json` file for heroku, that can be used to create an app on heroku from a GitHub repository.

Reviewer 2: Just one last suggestion:
```suggestion
This project comes with an `app.json` file, which can be used to create an app on Heroku from a GitHub repository.
```
